packages/enhanced/test/configCases/container/module-federation-with-shareScope/index.js

```diff
@@ -1,3 +1,15 @@
+if (globalThis.__FEDERATION__) {
```

Comment:
you can use the global method's resetFederationGlobalInfo in beforeEach hook

Reply:
The unit test env is mocked , and it's not easy to do this . I can do it in another pr